pkg/event/factory.go

```diff
@@ -297,17 +297,10 @@ func isValidAttribute(value interface{}) bool {
 	if value == nil {
 		return false
 	}
-	if _, ok := value.(string); ok {
-		return true
-	}
-	if _, ok := value.(float64); ok {
-		return true
-	}
-	if _, ok := value.(int); ok {
-		return true
-	}
-	if _, ok := value.(bool); ok {
+	switch value.(type) {
+	case string, float64, int, bool:
```

Comment:
Does this change work for float32, int8, which should be accepted?

Reply:
https://go.dev/tour/basics/11
should give all types?

Reply:
have covered all basic types now, please check 👍